pkg/sentry/identity/kubernetes/validator_test.go

```diff
@@ -0,0 +1,122 @@
+package kubernetes
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	kauthapi "k8s.io/api/authentication/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	fake "k8s.io/client-go/kubernetes/fake"
+	core "k8s.io/client-go/testing"
+)
+
+func TestValidate(t *testing.T) {
+	t.Run("invalid token", func(t *testing.T) {
+		fakeClient := &fake.Clientset{}
+		fakeClient.Fake.AddReactor(
+			"*",
+			"tokenreviews",
+			func(action core.Action) (bool, runtime.Object, error) {
+				switch action.GetVerb() {
+				case "create":
+				}
+				return true, &kauthapi.TokenReview{Status: kauthapi.TokenReviewStatus{Error: "bad token"}}, nil
```

Comment:
Technically, this return needs to be under `case "create"` because it should return the expected object only for create verb. for example,
```
                                switch action.GetVerb() {
				case "create":
                                        return true, &kauthapi.TokenReview{Status: kauthapi.TokenReviewStatus{Error: "bad token"}}, nil
				}
				return true, &kauthapi.TokenReview{},, nil
```

Can you change it? 

the other tests should have same pattern.

Reply:
done